admin/admin-filters-columns.php

```diff
@@ -454,7 +413,7 @@ public function ajax_update_term_rows() {
 	 * @param PLL_Language $language PLL_Language object.
 	 * @return string
 	 */
-	protected function get_flag_html( $language ) {
+	protected function get_flag_html( PLL_Language $language ): string {
```

Comment:
As discussed, we should internalize this method in `PLL_Language`. Some rework might be needed because `PLL_Language::get_flag_html()` already exists but doesn't serve the same prupose.
Let's address is in a separate PR.